packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.tsx

```diff
@@ -0,0 +1,77 @@
+import { Component, Prop, Element, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-usa-banner',
+  styleUrl: 'cbp-usa-banner.scss'
+})
+export class CbpUsaBanner {
+
+  @Element() host: HTMLElement;
+
+  /** When set, specifies that the banner is open */
+  @Prop({ reflect: true }) open: boolean;
+
+  // /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
+  // @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+  
+  // /** A custom event emitted when the banner link control is activated. */
+  handleClick() {
+    // e.preventDefault();
+    this.open = !this.open;
+    //set aria-expanded if it isn't comine with button
+  }
+
+  render() {
+    return (
+      <Host>
+         <section aria-label="Official website of the United States government">
+           <div class="cbp-banner__title">
+              <img src="../assets/images/us_flag_small.png" height="11" width="16" alt="usa logo" />
```

Comment:
can we inline this? This relative path is gonna bite us eventually. Here's the code:

```
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAALCAMAAABBPP0LAAAAG1BMVEUdM7EeNLIeM7HgQCDaPh/bPh/bPx/////bPyBEby41AAAAUElEQVQI123MNw4CABDEwD3jC/9/MQ1BQrgeOSkIqYe2o2FZtthXgQLgbHVMZdlsfUQFQnHtjP1+8BUhBDKOqtmfot6ojqPzR7TjdU+f6vkED+IDPhTBcMAAAAAASUVORK5CYII=" />
```

Just add the rest of the attributes back.

Reply:
updated to use cbp-icon